std/path.d

```diff
@@ -2893,7 +2893,7 @@ if (isConvertibleToString!R)
     `Exception` if the specified _base directory is not absolute.
 */
 string relativePath(CaseSensitive cs = CaseSensitive.osDefault)
-    (scope return string path, lazy string base = getcwd())
```

Comment:
ditto

Reply:
`relativePath` calls `asRelativePath` which takes input ranges, and scope inference fails there